k8s/configuration/secret.yaml

```diff
@@ -94,8 +94,9 @@ type: Opaque
 data:
   # Base64-encoded HTTPS private key
   cert.key: "<b64 encoded HTTPS private key>"
-  # Base64-encoded HTTPS Signed Certificate or Certificate Chain
-  cert.pem: "<b64 encoded HTTPS Signed Certificate or Certificate Chain>"
+  # Base64-encoded HTTPS certificate chain
+  # including all intermediate certificates and the actual certificate
+  cert.pem: "<b64 encoded HTTPS certificate chain"
```

Comment:
And since we are specifying that, we can also specify the order in which the chain can be formed.
Info here: https://www.digicert.com/ssl-support/pem-ssl-creation.htm under `Creating a .pem with the Entire SSL Certificate Trust Chain`